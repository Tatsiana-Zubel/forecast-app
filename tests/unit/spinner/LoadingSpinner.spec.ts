import { shallowMount } from '@vue/test-utils'
import LoadingSpinner from '@/components/spinner/LoadingSpinner.vue'

describe('LoadingSpinner.vue', () => {
  const wrapper = shallowMount(LoadingSpinner)

  it('has "wrapper" class', () => {
    const wrapperClass = wrapper.find('.wrapper')
    expect(wrapperClass.exists())
  })

  it('has "spinner" class', () => {
    const wrapperClass = wrapper.find('.spinner')
    expect(wrapperClass.exists())
  })

  it('should match snapshot', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
