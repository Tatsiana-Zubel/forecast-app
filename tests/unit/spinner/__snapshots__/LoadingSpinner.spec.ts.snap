// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingSpinner.vue should match snapshot 1`] = `
<div
  class="wrapper"
>
  <div
    class="spinner"
  />
</div>
`;
